recipes/questionary/LICENSE.txt

```diff
@@ -0,0 +1,19 @@
+Copyright 2020 Tom Bocklisch and contributors
```

Comment:
You can remove this file since it's in the source distribution

Reply:
```suggestion
```

Reply:
Just to clarify, I meant that you an remove the entire license file from within the recipe directory of this package.

Reply:
Sorry, I don't think @apeltzer or I realised this at first - now deleted! 🗑️